features/admin.rules.v1/components/rule-conditions.scss

```
@@ -20,6 +20,26 @@
     .MuiAutocomplete-moreItemsAvailableMessage {
         color: #666666;
         font-style: italic;
-        padding: 12px 15px 5px;
+        padding: 5px 15px 5px;
+        pointer-events: none;
+    }
+
+    .MuiAutocomplete-clearFilterLink {
+        font-style: italic;
+        padding: 5px 0;
+        cursor: pointer;
+
+        > .icon {
```

we don't need `>` in scss IINM.

```suggestion
        .icon {
```

---

Well in this case. Not necessary.